plugins/destination/bigquery/client/migrate.go

```diff
@@ -20,35 +20,35 @@ const (
 
 // Migrate tables. It is the responsibility of the CLI of the client to lock before running migrations.
 func (c *Client) Migrate(ctx context.Context, tables schema.Tables) error {
-	client, err := c.bqClient(ctx)
-	if err != nil {
-		return fmt.Errorf("failed to create client: %w", err)
-	}
-	eg, gctx := errgroup.WithContext(ctx)
+	// client, err := c.bqClient(ctx)
+	// if err != nil {
+	// 	return fmt.Errorf("failed to create client: %w", err)
+	// }
+	eg, gctx := errgroup.WithContext(context.Background())
```

Comment:
I think this should use the given `ctx`, no? My understanding is that only the BQ client needs to use the background context.

Reply:
@yevgenypats This still needs to be updated I believe